package.json

```diff
@@ -7,7 +7,7 @@
     "serve:prod": "concurrently --kill-others --names \"dev-server,ui,docs\" -c \"blue, red,green\" \"node server\" \"yarn --cwd ui build --watch --mode production --report\" \"yarn --cwd docs serve --mode production --report\"",
     "build:ui": "yarn --cwd ui build",
     "build:ui:dev": "cd ui && yarn build --mode development",
-    "build:docs": "yarn --cwd docs build --mode production --report",
+    "build:docs": "NODE_ENV=production; yarn --cwd docs build --mode production --report",
```

Comment:
`--mode production` _should_ accomplish the same thing as `NODE_ENV=production`, but I wasn't getting consistent results. Maybe someone more familiar with yarn can shed some light on that.

Reply:
`--mode production` is actually a Vue specific option. It simply tells Vue (through yarn) to build everything in production mode (minified, …)

Reply:
ah ok that makes sense!